src/components/Graph.js

```diff
@@ -0,0 +1,49 @@
+import { Circle } from 'rc-progress'
+
+const Graph = (props) => {
+  let count = props.tasks.count
+  let length = props.tasks.list.length
+  let countComplete = props.tasks.complete
+  let countTodo = props.tasks.todo
+  let countInProgress = props.tasks.inProgress
+
+  let percentComplete = length > 0 ? countComplete/count*100 : 0
+  let percentTodo = length > 0 ? countTodo/count*100 : 0
+  let percentInProgress = length > 0 ? countInProgress/count*100 : 0
+  return (
+    <div className='container bordered-container'>
+      <div className='col-md-4 graph-border'>
+        <div className='text-center'>
+          <p className='center-text-horizontal graphOne graphNumber' > {countComplete} </p>
+          <p className='center-text-horizontal graphText' > Complete </p>
+        </div>
+        <p className='percent'>{Math.floor(percentComplete)}%</p>
+        <div className='col-md-6 col-centered'>
+          <Circle percent={ percentComplete } strokeWidth='10' strokeColor='#3EC556' />
+        </div>
+      </div>
```

Comment:
Each one of these graphs appear to be treated the same. The only differences seem to be data driven:
count, %, title, color.
It seems reasonable to have a containing component which holds all 3 graphs, but it could in turn cycle over an array of things that need graphing and render a text component giving it the above mentioned props.